### ISSUE TITLE
Red meat allergy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9950]
**Preferred term label:**
Red meat allergy

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0410320

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8344025/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
THere are two terms

This one matches to the article you cite:
[Alpha-gal allergy HP:0410319](https://hpo.jax.org/app/browse/term/HP:0410319)
and there is this one
[Meat allergen allergy HP:0410330](https://hpo.jax.org/app/browse/term/HP:0410330)

Please reopen if they are not sufficient
